src/providers/Project/projectHooks.ts

```diff
@@ -22,7 +22,13 @@ function cloneProject(client: any, project: Project) {
   const localProject = createLocalProject(
     project.id,
     project.seed,
-    project.accounts.map(acc => acc.draftCode),
+    project.accounts.map(acc => acc.state),
+
+    project.contracts.map(con => ({
+      code: con.code,
+      title: con.title,
+      accountIndex: con.accountIndex
+    })),
```

Comment:
```suggestion
    project.contracts.map(({ code, title, accountIndex }) => ({ code, title, accountIndex })),
```

Reply:
Maybe alter scripts and transaction templates in the same manner? 🤔

Reply:
yes

Reply:
this commit fixed this: https://github.com/onflow/flow-playground/pull/177/commits/b7acfcc9e8149daad022f7bb47aac1296f7a2eca